src/lib/steps/setupRedirects.js

```diff
@@ -47,10 +47,16 @@ const setupRedirects = async (publishPath) => {
   const staticRedirects = nextRedirects.filter(({ route }) => !isDynamicRoute(removeFileExtension(route)))
   const dynamicRedirects = nextRedirects.filter(({ route }) => isDynamicRoute(removeFileExtension(route)))
 
-  // Add next/image redirect to our image function
+  // Add necessary next/image redirects for our image function
   dynamicRedirects.push({
     route: '/_next/image*  url=:url w=:width q=:quality',
```

Comment:
Rather than using a redirect, maybe we should emulate the URL structure of one of the built-in loaders (Cloudinary and Imgix) and then set that as the loader, with our root path. https://nextjs.org/docs/basic-features/image-optimization#loader  It looks like that's what Vercel does for sites they host: they switch the loader to Cloudinary, with `assets.vercel.com` as the path.